src/components/missions.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
- [ ] Let's try not to disable linters as they help us write cleaner code.

Reply:
Hi @Ghiftee 

removed disable at this [commit](https://github.com/Lucash2022/React-Group-Project/pull/34/commits/f06f06f581e8f5af40747f8576bcf1b55b42106c)